zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/usermanagement/UserControllerTest.java

```diff
@@ -310,6 +312,35 @@ void deleteUserShouldReturnNoContent() {
     verify(userServices, times(1)).deleteUser(key);
   }
 
+  @Test
+  void updateUserShouldReturnNoContent() {
```

Comment:
🔧 Can we also add a test in here to check the 409 conflict? :)

Reply:
The change here is just documenting the behaviour of the existing create endpoint, which already has a test here: https://github.com/camunda/camunda/blob/8025a375dfafb7a0b84ba4f8853e13acffb55089/zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/usermanagement/UserControllerTest.java#L82 
It's not quite right because we actually throw a different exception, but I can fix that in a follow up. The unit tests are a bit too shallow for my taste, which makes it hard to prevent this kind of divergence.

Reply:
Thats fine with me :)